bitbots_blackboard/CMakeLists.txt

```diff
@@ -1,7 +1,19 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.0.2)
 project(bitbots_blackboard)
 
-find_package(catkin REQUIRED COMPONENTS rospy bitbots_docs)
+find_package(catkin REQUIRED COMPONENTS
```

Comment:
Why is this added here?

Reply:
As I added `std_srvs` in the ball filter and here, I noticed other things are missing too.